src/AppHosts/SuperMemoAssistant/CollectionSelectionWindow.xaml.cs

```diff
@@ -246,34 +249,59 @@ private void ListBoxItem_MouseDoubleClick(object sender, MouseButtonEventArgs e)
       lbCollections.SelectFirstItem();
     }
 
+    private void TitleLink_RequestNavigate(object sender, RequestNavigateEventArgs e)
+    {
+      Process.Start(new ProcessStartInfo(e.Uri.AbsoluteUri));
+      e.Handled = true;
+    }
+
     private void ShowQuoteOfTheDay(object sender, RoutedEventArgs e)
     {
 
       var QuoteFile = SMAFileSystem.AppRootDir.CombineFile("quotes.csv");
 
       if (QuoteFile.Exists())
       {
-        var Lines = File.ReadAllLines(QuoteFile.FullPath);
-        var RandInt = new Random();
-        var RandomLineNumber = RandInt.Next(0, Lines.Length - 1);
-        var QuoteLine = Lines[RandomLineNumber];
-        var SplitQuoteLine = QuoteLine.Split('|');
-
-        // Pipe | separated file.
-        // Field 0 = quote text
-        // Field 1 = quote author
-        // Field 2 = quote URL
-        // Field 3 = quote title
-        if (!SplitQuoteLine[0].EndsWith(".")
-            && !SplitQuoteLine[0].EndsWith("!")
-            && !SplitQuoteLine[0].EndsWith("?"))
+        try
         {
-          SplitQuoteLine[0] += ".";
+          var Lines = File.ReadAllLines(QuoteFile.FullPath);
+
+          // Line 1 is .tsv heading
+          if (Lines.Length <= 1)
+          {
+            return;
+          }
+
+          var RandInt = new Random();
+          var RandomLineNumber = RandInt.Next(1, Lines.Length - 1);
+          var QuoteLine = Lines[RandomLineNumber];
+          var SplitQuoteLine = QuoteLine.Split('\t');
+          
+          // Tab separated file
+          // Field 0: Quote
+          // Field 1: Author
+          // Field 2: Url
+          // Field 3: Title
+          if (!SplitQuoteLine[0].EndsWith(".")
+              && !SplitQuoteLine[0].EndsWith("!")
+              && !SplitQuoteLine[0].EndsWith("?"))
+          {
+            SplitQuoteLine[0] += ".";
+          }
+
+          QuoteBodyTextBlock.Text = "\"" + SplitQuoteLine[0] + "\"";
```

Comment:
FYI, there is a neat feature in C# called string interpolation. It's not very useful in this case, but often it will make it easier to read strings with embedded variables. Either:
- `$"\"{SplitQuoteLine[0]}\""`
- `$@"""{SplitQuoteLine[0]}"""` (double "" equals escaped ")